tags/tag-operational-resilience/charter.md

```diff
@@ -1 +1,83 @@
+* **Resource Optimization:** The efficient use of computational, storage, and network resources to balance performance and cost. 
+* **Cost Efficiency:** Methodologies for optimizing expenditures without compromising system reliability or performance. 
+* **Energy Efficiency:** Focusing on and providing guidance for reducing the energy consumption of cloud native services.
+* **Performance:** Ensuring the speed, responsiveness, and stability of cloud native applications under various load conditions.
+* **Troubleshooting:** The systematic process of identifying, diagnosing, and resolving issues within a distributed system.
+* **Testing:** Verifying the reliability and resilience of applications
+* **Reliability:** The ability of a system to perform its required functions under stated conditions for a specified period of time.
+* **Day 2 Ops:** A focus on the post-deployment activities required to keep a system running smoothly, including maintenance, scaling, updating and upgrading.
+* **Chaos Engineering:** The practice of experimenting on a system to build confidence in its capability to withstand turbulent and unexpected conditions.
+* **Conformance Programs:** Relevant ownership and development of conformance programs to help end users adapt and choose platforms and tools that adhere to established operational resilience standards
+
+### Out of Scope
+
+* This TAG will collaborate with other TAGs on overlapping areas, but will not primarily focus on topics explicitly owned by different TAGs
+* Form an umbrella organization beyond the CNCF
+* Evaluate individual company infrastructures and provide advice
+* Focus outside of cloud native technologies, according to the [CNCF Cloud Native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md)
+
+
+## Deliverables to TOC
+
+The anticipated deliverables for TAG Operational Resilience include:
+
+* **Subprojects:** Ongoing services or programs requiring stewardship from the TAG. Examples include:
+  * Production Readiness Assessments
+  * Resource Optimization and Green Reviews
+* **Initiatives:** Time-bound, objective-focused units of work submitted to the TOC. Examples may include:
+  * Specific assessments for Resource Optimization or Cost Efficiency best practices
+  * Guidelines for performance testing best practices
+* **Whitepapers and Guides:** In-depth documents that provide frameworks and guidelines consistent with advancing practices in the TAG's scope. These serve projects, other TAGs, and the TOC. 
+* **Landscape Analysis:** A curated view of the projects, tools, and standards within the operational resilience ecosystem, similar to the Cloud Native Landscape, to identify gaps and opportunities.
+* **Knowledge Gap Identification:** Reports and presentations to the TOC on identified gaps within the cloud native ecosystem related to operational resilience. 
```

Comment:
Just knowledge gaps? or knowledge, tooling, guidance, and capability gaps?

Reply:
I think it makes sense to move this up to the Mission statement and elaborate on it there.

Reply:
@halcyondude can you move this up to the mission statement as discussed